src/1a_prep_MPA.jl

```diff
@@ -1,189 +1,298 @@
+            mask(
+                crop(gbr_benthic; to=regions_4326[reg_idx_4326, :]);
+                with=regions_4326[reg_idx_4326, :]
+            )
+        )
+
+        write(
+            benthic_fn,
+            resample(
+                target_benthic;
+                to=bathy_gda2020
+            )
+        )
+
+        gbr_benthic = nothing
         target_benthic = nothing
-        GC.gc()
+        force_gc_cleanup()
     end
 
-    if !isfile(joinpath(MPA_OUTPUT_DIR, "$(reg)_geomorphic.tif"))
-        bathy_gda2020_path = joinpath(MPA_OUTPUT_DIR, "$(reg)_bathy.tif")
-        bathy_gda2020 = Raster(bathy_gda2020_path; crs=EPSG(7844), lazy=true)
-
-        # Trim raster to region of interest
-        target_geomorphic = Rasters.trim(mask(gbr_geomorphic; with=regions_4326[reg_idx_4326, :]))
-        # Reproject raster to GDA 2020
-        target_geomorphic = resample(target_geomorphic; to=bathy_gda2020)
+    geomorphic_fn = joinpath(MPA_OUTPUT_DIR, "$(reg)_geomorphic.tif")
+    if !isfile(geomorphic_fn)
+        gbr_morphic_path = "$(MPA_DATA_DIR)/geomorphic/GBR10 GBRMP Geomorphic.tif"
+        target_geomorphic = Raster(gbr_morphic_path, mappedcrs=EPSG_4326)
+
+        target_geomorphic = Rasters.trim(
+            mask(
+                crop(target_geomorphic; to=regions_4326[reg_idx_4326, :]);
+                with=regions_4326[reg_idx_4326, :]
+            )
+        )
+
+        write(
+            geomorphic_fn,
+            resample(
+                target_geomorphic;
+                to=bathy_gda2020
+            )
+        )
 
-        write(joinpath(MPA_OUTPUT_DIR, "$(reg)_geomorphic.tif"), target_geomorphic; force=true)
-
-        bathy_gda2020 = nothing
         target_geomorphic = nothing
-        GC.gc()
+        force_gc_cleanup()
     end
 
-    if !isfile(joinpath(MPA_OUTPUT_DIR, "$(reg)_waves_Hs.tif"))
+    target_waves_Hs_fn = joinpath(MPA_OUTPUT_DIR, "$(reg)_waves_Hs.tif")
+    if !isfile(target_waves_Hs_fn)
+        # Use bathy dataset as a template for writing netCDF data to geotiff
         src_bathy_path = first(glob("*.tif", joinpath(MPA_DATA_DIR, "bathy", reg)))
         src_bathy = Raster(src_bathy_path, mappedcrs=EPSG(4326), lazy=true)
 
-        target_waves_Hs_path = first(glob("*.nc", joinpath(WAVE_DATA_DIR, "Hs", reg)))
-        target_waves_Hs = Raster(target_waves_Hs_path, key=:Hs90, crs=crs(src_bathy), mappedcrs=EPSG(4326), lazy=true)
+        waves_Hs_path = first(glob("*.nc", joinpath(WAVE_DATA_DIR, "Hs", reg)))
 
-        # Extend bounds of wave data to match bathymetry if needed
-        if size(src_bathy) !== size(target_waves_Hs)
-            target_waves_Hs = extend(crop(target_waves_Hs; to=src_bathy); to=AG.extent(src_bathy))
-            @assert size(src_bathy) == size(target_waves_Hs)
+        # Have to load netCDF data into memory to allow missing value replacement
+        waves_Hs = Raster(
+            waves_Hs_path,
+            key=:Hs90,
+            crs=EPSG_7856,
```

Comment:
I thought the wave files are sort of in UTM zone ~55 crs mapping. This probably doesn't matter if the data is just copied over to src_bathy manually anyway.